package ua.goit.module.tenth.task5;

public class ThirdException extends Exception{
}
